scripts/xslt/cdrh_to_html/lib/html_formatting.xsl

```diff
@@ -587,7 +585,7 @@
 <!--              SUPER AND SUB SCRIPT                -->
 <!-- ================================================ -->
 
-  <xsl:template match="p[@rend='superscript'] | p[@rend='sup'] | hi[@rend='super'] | hi[@rend='superscript']">
+ <xsl:template match="p[@rend='superscript'] | p[@rend='sup'] | hi[@rend='super'] | hi[@rend='superscript']" priority="2">
```

Comment:
same here in terms of checking if this priority will potentially interfere with other overrides.  I am not very familiar with the lewis and clark / oscys HTML generation, so you might be able to verify this better than me.  I looked at their p5 xsl files quickly and didn't see templates matching on these types of rends, so hopefully it's all good!